apps/prerender/src/components/Profile.tsx

```diff
@@ -138,23 +138,27 @@ const Profile: FC<ProfileProps> = ({ profile, publications }) => {
         <hr />
       </header>
       <div data-testid="profile-feed">
-        {publications?.map((publication) => {
-          const { __typename } = publication;
-          return (
-            <div
-              key={
-                __typename === 'Mirror'
-                  ? publication.mirrorOf.id
-                  : publication.id
-              }
-            >
-              <SinglePublication publication={publication} />
-            </div>
-          );
-        })}
+        {publications ? (
```

Comment:
publications can be an `[]` empty array and it would still go with for loop? 
```suggestion
        {publications.length ? (
```
why'd we need this?

Reply:
yeah but if we do the length check then publications can be undefined hence we miught need to add `publications && publications.length > 0 ?` which @bigint might contest?